common/changes/office-ui-fabric-react/slider-number-format_2019-01-04-19-23.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Allows a custom value format function to be passed to the Slider component",
```

Comment:
nit: could you please change the comment to start with _'Slider: .....'_? Makes it easier to scan through the release notes and see what components have changed. Thanks!

Reply:
Sure thing! Done.